JS.js

```diff
@@ -5,14 +5,19 @@ let highscore = 0;
 const displayMessage = function (message) {
   document.querySelector('.message').textContent = message;
 };
+
+
 document.querySelector('.check').addEventListener('click', function () {
   const guess = Number(document.querySelector('.guess').value);
-  console.log(guess, typeof guess);
+
+  
   if (!guess) 
   {
+
     document.querySelector('.message').textContent = ' No number!';
```

Comment:
you already have a function to desplay the message, you can use it here displayMessage()